ui/components.d.ts

```diff
@@ -52,11 +54,13 @@ declare module 'vue' {
     NSelect: typeof import('naive-ui')['NSelect']
     NSpace: typeof import('naive-ui')['NSpace']
     NSpin: typeof import('naive-ui')['NSpin']
+    NTable: typeof import('naive-ui')['NTable']
     NTabPane: typeof import('naive-ui')['NTabPane']
     NTabs: typeof import('naive-ui')['NTabs']
     NTag: typeof import('naive-ui')['NTag']
     NText: typeof import('naive-ui')['NText']
     NThing: typeof import('naive-ui')['NThing']
+    NTooltip: typeof import('naive-ui')['NTooltip']
     NTree: typeof import('naive-ui')['NTree']
     NUpload: typeof import('naive-ui')['NUpload']
     NUploadDragger: typeof import('naive-ui')['NUploadDragger']
```

Comment:
The primary difference is between lines 27 and 30 of file 'naive-buefy'. Instead of `import` statement like this:

```javascript
declare namespace 'vue-i18n'
```

you can write it directly without `declare`: 

```javascript
import { TOptionsWithDefault } from "naive-buefy"
export type NLocale = TOptionsWithDefault

I would suggest changing these two lines to the direct declaration because it makes the code more readable.

If no issues have been found otherwise, then there are no further suggestions required.